src/grabbing.c

```diff
@@ -224,11 +224,16 @@ void grabbing_xinput_grab_start(Grabber * self) {
 			XIEventMask mask = {
 			XIAllDevices, sizeof(mask_data), mask_data };
 
-			XIGrabModifiers mods = {
-			XIAnyModifier };
+			XIGrabModifiers mods[4] = {
+				{0, 0}, // no modifiers
+				{LockMask, 0}, // Caps lock
+				{Mod2Mask, 0}, // Num lock
+				{LockMask | Mod2Mask, 0} // Caps & Num lock
+			};
+
```

Comment:
I need to think about this... I really like to be able to do gestures even if I'm pressing some modifier keys like shift and win...  Maybe we can think about a new command line argument... what do you think?

Reply:
It could be a command line flag as a quick workaround. Although a better way I think would be to allow specifying modifiers mask for each gesture.